src/test/java/gov/acwi/wqp/etl/monitoringLocation/TransformMonitoringLocationIT.java

```diff
@@ -0,0 +1,102 @@
+package gov.acwi.wqp.etl.monitoringLocation;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+import java.nio.charset.Charset;
+import java.sql.SQLException;
+
+import javax.annotation.PostConstruct;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.springframework.batch.core.ExitStatus;
+import org.springframework.batch.core.Job;
+import org.springframework.batch.core.JobExecution;
+import org.springframework.batch.core.job.flow.Flow;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.core.io.support.EncodedResource;
+import org.springframework.jdbc.datasource.init.ScriptException;
+import org.springframework.jdbc.datasource.init.ScriptUtils;
+
+import com.github.springtestdbunit.annotation.DatabaseSetup;
+import com.github.springtestdbunit.annotation.ExpectedDatabase;
+import com.github.springtestdbunit.assertion.DatabaseAssertionMode;
+
+import gov.acwi.wqp.etl.BiodataBaseFlowIT;
+import gov.acwi.wqp.etl.monitoringLocation.index.BuildMonitoringLocationIndexesFlowIT;
+import gov.acwi.wqp.etl.monitoringLocation.table.SetupMonitoringLocationSwapTableFlowIT;
+
+public class TransformMonitoringLocationIT extends BiodataBaseFlowIT {
+
+	@Autowired
+	@Qualifier("monitoringLocationFlow")
+	private Flow monitoringLocationFlow;
+	
+	public static final String EXPECTED_DATABASE_QUERY_INDEX = "select tablename, indexname, indexdef from pg_indexes where tablename='station_swap_biodata'";
```

Comment:
The front portion of these two constants are now part of BaseFlowIT.

Reply:
Updated